cmake/SelectPcsx2Plugins.cmake

```diff
@@ -5,7 +5,7 @@ set(msg_dep_common_libs "check these libraries -> wxWidgets (>=2.8.10), aio")
 set(msg_dep_pcsx2       "check these libraries -> wxWidgets (>=2.8.10), gtk2 (>=2.16), zlib (>=1.2.4), pcsx2 common libs")
 set(msg_dep_cdvdiso     "check these libraries -> bzip2 (>=1.0.5), gtk2 (>=2.16)")
 set(msg_dep_zerogs      "check these libraries -> glew (>=1.6), opengl, X11, nvidia-cg-toolkit (>=2.1)")
-set(msg_dep_gsdx        "check these libraries -> opengl, png++, X11")
+set(msg_dep_gsdx        "check these libraries -> opengl, png (>=1.2), X11")
```

Comment:
I'm not sure I understand you.

I tested against libpng 1.6 on Windows and Linux, and it compiled fine on Travis CI which has libpng 1.2. Most (if not all) distros will be using libpng 1.2 or later.